torch/_tensor_docs.py

```diff
@@ -2418,23 +2418,23 @@ def callable(a, b) -> number
 
 add_docstr_all('scatter_add_',
                r"""
-scatter_add_(dim, index, other) -> Tensor
+scatter_add_(dim, index, src) -> Tensor
```

Comment:
"scatter_add" is also wrong... https://pytorch.org/docs/stable/tensors.html#torch.Tensor.scatter_add (it takes src but the docs lists it as "source")

Reply:
fixed below :)

Reply:
Oho, I didn't read the entire PR. Thanks!! (and feel free to ping again for review earlier in the future, the notification for this escaped my radar)